src/Teams/Teams.csproj

```diff
@@ -7,6 +7,13 @@
     <DockerComposeProjectPath>..\docker-compose.dcproj</DockerComposeProjectPath>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
I can't see those files in the pull-request so it means that they will not be presented in this code base so it means that lines in csproj is useless and should be removed